src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -12,14 +14,31 @@ import { NOT_AVAILABLE } from './utils'
 import TxShareButton from './TxShareButton'
 import TxInfoMultiSend from './TxInfoMultiSend'
 import DelegateCallWarning from './DelegateCallWarning'
-import { generateSignaturesFromTxConfirmations } from 'src/logic/safe/safeTxSigner'
-import { makeConfirmation } from 'src/logic/safe/store/models/confirmation'
-import { TxDataRow } from './TxDataRow'
+import { TxDataRow } from 'src/routes/safe/components/Transactions/TxList/TxDataRow'
+
+const StyledButtonLink = styled(ButtonLink)`
+  padding-left: 0;
+
+  & > p {
+    margin-left: 0;
+  }
+`
+
+const CollapsibleSection = styled.div<{ show: boolean }>`
+  max-height: ${({ show }) => (show ? '500px' : '0px')}; // We need to set a fixed height for the animation to work
+  overflow: hidden;
+  transition: ${({ show }) => (show ? 'max-height 0.4s ease-in-out' : 'max-height 0.2s cubic-bezier(0, 1, 0, 1)')};
+`
 
 type Props = { txDetails: ExpandedTxDetails }
 
 export const TxSummary = ({ txDetails }: Props): ReactElement => {
   const { txHash, detailedExecutionInfo, executedAt, txData, txInfo } = txDetails
+  const [expanded, setExpanded] = useState(false)
+
+  const toggleExpanded = () => {
```

Comment:
Does this intefere with delegate call warnings?

Reply:
No. this is to expand solely the "Advanced Details"